frontend/docsettings.lua

```diff
@@ -50,7 +51,7 @@ local function buildCandidates(list)
     end
 
     -- MRU sort, tie breaker is insertion order (higher priority locations were inserted first).
-    -- Iff a primary/backup pair of file both exist, of the two of them, the primary one *always* has priority,
+    -- If a primary/backup pair of file both exist, of the two of them, the primary one *always* has priority,
```

Comment:
Iff is a shortcut for "if and only if", not a typo, FWIW ;).

Reply:
I still remember when my high school math teacher talked about how clever he thought the English "iff" was.

Reply:
Didn't know that, thanks, have found more iff's in your (and only your) code.

Reply:
Yeah, it's a weird little abbreviation ;o).